Vagrantfile

```diff
@@ -15,6 +15,14 @@ Vagrant.configure('2') do |config|
     vb.name = 'sumologic-openshift-images'
   end
 
+  config.vm.provider "qemu" do |qe, override|
+    override.vm.box = "perk/ubuntu-2204-arm64"
+    qe.gui = false
```

Comment:
Nice image but is it maintained? 😄

Reply:
There is no other 😂 
I'm thinking about creating my own to be honest as I cannot contact with the author 🙃